learn/advanced/chronological_filtering_sorting.md

```diff
@@ -0,0 +1,109 @@
+# Filtering and sorting with chronological values
+
+In this guide, you will learn about Meilisearch's approach to date and time values, how to prepare your dataset for indexing, and how to chronologically sort and filter search results.
+
+## Preparing your documents
+
+To filter and sort search results based on chronological values, your documents must have a numeric field containing a UNIX timestamp. In this videogames dataset, the release year is formatted as a timestamp:
```

Comment:
Pretty sure everyone reading would probably know what UNIX timestamps are, but maybe add a link for it? We do that for the ISO 8601 and RFC 3339 formats too.